src/docs/product/security/ip-ranges.mdx

```diff
@@ -14,12 +14,32 @@ apply to Self-hosted or Single Tenant.
 </markdown>
 </Alert>
```

Comment:
might want to convert this to the new note format:

<Note>
<markdown>

This page applies only to Sentry's SaaS product. It **does not**
apply to Self-Hosted or Single Tenant.

</markdown>
</Note>

Reply:
@PeloWriter: now that I spent more time reading this, _what_ new format are you referring to?